src/pages/new-chat/new-chat.html

```diff
@@ -10,9 +10,15 @@
 
 <ion-content class="new-chat">
   <ion-list class="users">
-    <button ion-item *ngFor="let user of users | async" class="user" (click)="addChat(user)">
+    <button ion-item *ngFor="let user of users | async | search:searchPattern:'profile.name'" class="user" (click)="addChat(user)">
```

Comment:
Over here: https://github.com/DAB0mB/Ionic2CLI-Meteor-WhatsApp/blob/47c805c8619936cfb4cefd90bee93cf7e8f6c462/src/pipes/search.ts#L8
I escaped the operation in case the value was `undefined`, if that's what you meant.